app/assets/javascripts/util.js

```diff
@@ -100,24 +100,16 @@ function updateArrayURLParameter(url, param, _paramVals) {
 }
 
 function getURLParameter(_name, _url) {
-    const url = _url || window.location.href;
-    const name = _name.replace(/[[\]]/g, "\\$&");
-    const regex = new RegExp("[?&]" + name + "(=([^&#]*)|&|#|$)");
-    const results = regex.exec(url);
-    if (!results) return null;
-    if (!results[2]) return "";
-    return decodeURIComponent(results[2].replace(/\+/g, " "));
+    const url = new URL(_url || window.location.href);
```

Comment:
```suggestion
    const url = new URL(_url ?? window.location.href);
```

@chvp I think this is never called with an empty string so we can used nullish coalescing here instead of the logical or?

If so, also change in the function below.

Reply:
Sound reasonable, if it was called with the empty string it was probably a bug that the `href` was used in that case.